roles/oVirt.datacenter-cleanup/tasks/main.yml

```diff
@@ -11,27 +14,29 @@
 - name: Remove Disks
   include: disks.yml
 
-- name: Find existing Storage Domains in Datacenter
-  ovirt_storage_domains_facts:
-    auth: "{{ ovirt_auth }}"
-    pattern: "datacenter={{ data_center_name }}"
-
-- name: Remove all Storage Domains except master
+- name: Remove all Storage Domains except master and hosted_storage if HE
   include: storages_pre.yml
 
 - name: Find existing clusters in Datacenter
-  ovirt_clusters_facts:
+  ovirt_cluster_facts:
     auth: "{{ ovirt_auth }}"
     pattern: "datacenter={{ data_center_name }}"
 
+- name: Remove hosts except from the one hosting the engine VM
+  include: remove_hosts_not_hosting_engine_vm.yml
```

Comment:
include is depracated: https://docs.ansible.com/ansible/2.5/user_guide/playbooks_reuse_includes.html